ui/src/screens/SystemStatusScreen/SystemStatusScreen.scss

```diff
@@ -1,15 +1,12 @@
-@import '~src/app/tables';
-@import 'app/mixins.scss';
-
-.StatusTable {
-  @extend .data-table;
-
-  tr td,
-  tr th {
-    vertical-align: middle;
-  }
-
-  td.text {
-    @include rtlSupportInvertedProp(padding, left, 0, null);
-  }
-}
+// @import '~src/app/tables';
+// @import 'app/mixins.scss';
+// .StatusTable {
+//   @extend .data-table;
+//   tr td,
+//   tr th {
+//     vertical-align: middle;
+//   }
+//   td.text {
+//     @include rtlSupportInvertedProp(padding, left, 0, null);
+//   }
+// }
```

Comment:
Wasn't able to track down what in this guy was breaking things, so I commented it out for now in the interest of getting this up with eyes on it. Not staying this way.